src/main/java/core/basesyntax/strategy/impl/BalanceOperation.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.StorageService;
+import core.basesyntax.service.impl.StorageServiceImpl;
+import core.basesyntax.strategy.Operation;
+
+public class BalanceOperation implements Operation {
+    private final StorageService storageService;
```

Comment:
It may be better to make StorageService a field of the Operation interface to avoid duplicating code in each implementation.

Reply:
you may be right, but I'm not sure, so I'll wait for the mentor's feedback

Reply:
As per your question regarding the code duplication of common field for all Handlers: interfaces can have only constants, so we can not do that. But if you really want to, you can add an abstract class in between interface and class, put this field and a constructor there, and remove from here and that will work, but I would say that this is unnecessary, so up to you